app/helpers/active_admin/allocation_chart_helper.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module ActiveAdmin
+  module AllocationChartHelper
+    class Status
+      EXP_IN_60_DAYS = 'exp-in-60-days'
+      EXP_IN_30_DAYS = 'exp-in-30-days'
+      EXPIRED = 'expired'
+      NOT_ALLOCATED = 'not-allocated'
+      ALLOCATED = 'allocated'
+    end
+
+    def allocation_status_for(last_allocation:)
+      return Status::NOT_ALLOCATED if last_allocation&.id.nil?
```

Comment:
is it necessary to check the `id` explicitly?

Reply:
Yes because some `last_allocation` values can be "empty" `Allocation` instances